components/prospects/ProfileInfo.tsx

```diff
@@ -31,7 +31,9 @@ function ProfileInfo({ prospect }) {
       <Typography sx={{ display: 'flex', justifyContent: 'flex-end' }}>
         <ProfileActions id={prospect.id} name={prospect.name} options={[
           { id: 0, label: 'Delete', action: deleteProfile },
-        ]} />
+        ]}
+        entityType={'prospect'}
```

Comment:
being a prop for profile actions, I think this needs to be indented one tab to the right. Same for row above